src/GridView/CardGrid.js

```diff
@@ -61,11 +62,25 @@ class CardGrid extends ConnectionComponent {
 		const attributeArray = [];
 		childEntity.Attributes.forEach((attribute) => {
 			attributeArray.push(attribute.Name);
-			attributeArray.push(attribute.Value)
+			attributeArray.push(attribute.Value);
 		});
 		return attributeArray;
 	}
 
+	static getNecessaryAttributes(entityType) {
+		const necessaryAttributes = [];
+		const attributeDefinition = attributeConfig[String(entityType.Name)];
+		if(!attributeDefinition) {
+			return null;
+		}
+		Object.entries(attributeDefinition).forEach(([attributeName, isNecessary]) => {
```

Comment:
An _object_ function? Can't you use a function defined on _attributeDefinition_?